linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -1,10 +1,23 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,ol7,rhv4
+prodtype: rhel6,rhel7,ol7,rhv4,sle12
 
-title: 'Use Only FIPS 140-2 Validated Ciphers'
+title: |-
+{{% if product == "sle12" %}}
+    Use Only DoD-approved encryption Ciphers
+{{% else %}}
+    Use Only FIPS 140-2 Validated Ciphers
+{{% endif %}}
 
 description: |-
+{{% if product == "sle12" %}}
+    Limit the ciphers to those algorithms which are DoD-approved.
+    Only Counter (CTR) mode is allowed inside the SSH daemon configuration.
+    The following line in <tt>/etc/ssh/sshd_config</tt>
+    demonstrates use of DoD-approved encryption ciphers:
+    <pre>Ciphers aes128-ctr,aes192-ctr,aes256-ctr</pre>
+    The man page <tt>sshd_config(5)</tt> contains a list of supported ciphers.
+{{% else %}}
```

Comment:
Nack on this change. FIPS is mandated by Federal Law. This change is not needed or desired.